README.md

```diff
@@ -204,28 +204,22 @@ The following are tutorials on KubeLinter written by users. If you have one that
 * [How to add a new check in KubeLinter?](https://www.psaggu.com/upstream-contribution/2021/08/17/notes.html) by Priyanka Saggu.
 * [Extending kube-linter To Build A Custom Template](https://github.com/garethahealy/kubelinter-extending-blog) by [Gareth Healy](https://github.com/garethahealy).
 
-## Community
-
-If you would like to engage with the KubeLinter community, including maintainers and other users, you can join the Slack workspace [here](https://join.slack.com/t/kube-linter/shared_invite/zt-kla9qvyo-Tk~wynTSbr9EE3AjHcv4BQ).
-
-To contribute, check out our [contributing guide](./CONTRIBUTING.md).
-
-As a reminder, all participation in the KubeLinter community is governed by our [code of conduct](./CODE_OF_CONDUCT.md).
+## LICENSE
 
-## WARNING: Alpha release
+KubeLinter is licensed under the [Apache License 2.0](./LICENSE).
 
-KubeLinter is at an early stage of development. There may be breaking changes in
-the future to the command usage, flags, and configuration file formats. However,
-we encourage you to use KubeLinter to test your environment YAML files, see what
-breaks, and [contribute](./CONTRIBUTING.md).
+## Community
 
-## LICENSE
+If you would like to engage with the KubeLinter community, including maintainers and other users, you can join the Slack workspace [here](https://join.slack.com/t/kube-linter/shared_invite/zt-kla9qvyo-Tk~wynTSbr9EE3AjHcv4BQ).
 
-KubeLinter is licensed under the [Apache License 2.0](./LICENSE).
+There may be breaking changes in the future to the command usage, flags, and configuration
+file formats. However, we encourage you to use KubeLinter to test your environment YAML
+files, see what breaks, and [contribute](./CONTRIBUTING.md).
```

Comment:
Would it make sense to have a backward compatibility sub-section for that piece ?

Reply:
Good thought. I guess at this point we don't really have clear backward compatibility guidelines nor guarantees, so maybe it's a bit too early to pull this out into a separate section? No strong opinion here, open for suggestions.

Reply:
That was food for thought. Adding a dedicated section will also make the bottom block bigger and might make the message less catchy. Things can be left in their current state for now.